src/Compilers/CSharp/Portable/Syntax/LiteralExpressionSyntax.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    public partial class SyntaxFactory
+    {
+        /// <summary>Creates a new LiteralExpressionSyntax instance.</summary>
+        public static LiteralExpressionSyntax LiteralExpression(SyntaxKind kind)
```

Comment:
I didn't understand why these methods moved out of the generated file. Could you please clarify?

Reply:
they're not getting generated autoamtically as there isn't a 1:1 correspondance with node-type and node-kind.  note: we will discuss this at the API review.  I personally think we don't need/want the 1:1 correspondence.